kubernetes/kubernetes.go

```diff
@@ -186,14 +185,22 @@ func (ka KubeAgentConfig) collectMetrics(
 	// get raw Heapster metric sample
 	hme, err := rawClient.GetRawEndPoint("heapster-metrics-export", metricSampleDir, config.HeapsterURL)
 	if err != nil {
-		return fmt.Errorf("error retrieving Raw Heapster metrics: %s", err)
+		return fmt.Errorf("unable to retrieve raw heapster metrics: %s", err)
 	}
 
 	defer util.SafeClose(hme.Close, &rerr)
 
-	err = handleBaselineHeapsterMetrics(msd, baselineMetricSample, hme.Name())
+	baselineMetricSample, err := filepath.Glob(path.Dir(config.msExportDirectory.Name()) + "/baseline-metrics-export*")
+
 	if err != nil {
-		return err
+		log.Printf("Error encountered reading export directory: %v", err)
+	}
+
+	if len(baselineMetricSample[0]) >= 1 {
```

Comment:
filepath.Glob returns a slice of matches. here we're checking the length of the first match. do we intend to do that? is there a case where the first match is an empty string? or is it meant to be checking the number of matches instead?